src/testWithSpringBoot_3_0/java/org/openrewrite/java/spring/boot3/AddMappingPathsWithTrailingSlashTest.java

```diff
@@ -0,0 +1,199 @@
+package org.openrewrite.java.spring.boot3;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class AddMappingPathsWithTrailingSlashTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion().classpath("spring-beans", "spring-context", "spring-boot", "spring-security", "spring-web", "spring-core"))
+          .recipe(new AddMappingPathsWithTrailingSlash());
+    }
+
+    @Test
+    void simpleCase() {
+        rewriteRun(
+          java(
+            """
+              import org.springframework.web.bind.annotation.*;
+
+              @RestController
+              public class ExampleController {
+
+                  @GetMapping("/get")
+                  public String getExample() {
+                      return "This is a GET example.";
+                  }
+              }
+              """,
+            """
+              import org.springframework.web.bind.annotation.*;
+
+              @RestController
+              public class ExampleController {
+
+                  @GetMapping(value = {"/get", "/get/"})
```

Comment:
When there is only one argument to an annotation _and_ the annotation type has a field named `value`, the argument is understood by the java compiler to be an argument to `value`. 

Whenever there is more than one argument, or there is no field on the annotation type named `value`, specifying the name of the annotation field becomes mandatory.

In this case since the "before" argument did not need `value =` I would not expect that `value =` would be introduced

Reply:
Good one, fixed.